tests/all/integration/api/helpers/test_scheduled_jobs.py

```diff
@@ -0,0 +1,29 @@
+import datetime
+
+from app import current_app as app, db
+from app.factories.event_invoice import EventInvoiceFactory
+from app.models.event_invoice import EventInvoice
+from app.api.helpers.scheduled_jobs import event_invoices_mark_due
+
+from tests.all.integration.utils import OpenEventTestCase
+
+
+class TestScheduledJobs(OpenEventTestCase):
+
+    def test_event_invoices_mark_due(self):
+        """Method to test marking of event invoices as due"""
+
+        with app.test_request_context():
+            event_invoice = EventInvoiceFactory(event__ends_at=datetime.datetime(2019, 7, 20))
+            db.session.commit()
+            event_invoice_id = event_invoice.id
+            event_invoices_mark_due()
+            status = EventInvoice.query.get(event_invoice_id).status
+            self.assertEqual(status, "due")
+
+            event_invoice = EventInvoiceFactory(status="paid")
+            db.session.commit()
+            event_invoice_id = event_invoice.id
+            event_invoices_mark_due()
+            status = EventInvoice.query.get(event_invoice_id).status
+            self.assertNotEqual(status, "due")
```

Comment:
Added the test case. @iamareebjamal

Reply:
There's one case missing for upcoming events with `(EventInvoice.created_at + datetime.timedelta(days=30) > datetime.datetime.now()`

Also, commit each one and run `event_invoices_mark_due()` once and verify the 3 invoice status

Reply:
This test seems to fail. Created a future datetime object and passed it into the `created_at` attribute.